<launch>

  <arg name="extNodeName" default=""/>
  <arg name="camTopicName" default="/cam0"/>


  <node name="visualizer" ns="rcars" pkg="rcars_visualization" type="rcars_image_visualization_node" output="screen">
    <param name="extNodeName" value="$(arg extNodeName)" />
    <param name="camTopicName" value="$(arg camTopicName)" />
  </node>

  <node name="tf_publisher" ns="rcars" pkg="rcars_visualization" type="rcars_tf_publisher_node" output="screen"> 
    <param name="align_workspace" value="false"/>
  	<param name="frame_rcars" value="rcars_tag54_dyn" />
  	<param name="frame_align" value="reference_tag" />
  	<param name="useExternalCalibration" value="false"/>
  	<param name="WrWTX" value="1.91" />
    <param name="WrWTY" value="-0.6" />
    <param name="WrWTZ" value="2.17" />
    <param name="qWTX" value="-1.57079632679" />    
    <param name="qWTY" value="3.14159265359" />
    <param name="qWTZ" value="0.0" />
  </node>
  
  <node name="rviz" pkg="rviz" type="rviz" respawn="false" args="-d $(find rcars_estimator)/config/debug.rviz" />
  
</launch>